.github/workflows/linux.yml

```
@@ -120,7 +115,8 @@ jobs:
           make dist
       - name: Update version
         if: |
-          github.ref == 'refs/heads/master'
+          !startsWith(github.ref, 'refs/tags/') &&
+          !startsWith(github.ref, 'refs/heads/maintenance/')
```

Or we keep using `vXX.YY-maintenance`:

```suggestion
          !endsWith(github.ref, '-maintenance')
```

---

You mean we need this modification if we keep using `vXX.YY-maintenance` branch?
If we use `maintenace/XX.YY` branch, is this modification needless?

I will use `maintenace/XX.YY` branch.

---

OK.